package.json

```diff
@@ -24,8 +24,9 @@
   },
   "dependencies": {
     "@aws-quickstart/eks-blueprints": "^1.11.3",
-    "constructs": "^10.0.0",
     "aws-cdk": "2.91.0",
+    "aws-sdk": "^2.1455.0",
```

Comment:
why is sdk needed?

Reply:
for getting ssm securestring cdk-context using SDK